kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -47,6 +47,7 @@ function _contentState(data) {
   } else {
     progress = data.progress_fraction;
   }
+  const assessmentMetaData = data.asessmentmetadata || {};
```

Comment:
Note: this needs to be updated to reflect the fact that the assessmentmetadata property will be an Array (but it *should* only be one item long) so it should be:

```const assessmentMetaData = (data.assessmentmetadata || [])[0];